fireplace/dsl/random_picker.py

```diff
@@ -1,79 +1,97 @@
 import random
 from hearthstone.enums import CardType, Race, Rarity
 from .lazynum import LazyValue
+from copy import copy
```

Comment:
Mind your import ordering. The copy import should be before the hearthstone import.